pkg/runner/runner.go

```diff
@@ -17,6 +19,11 @@ import (
 	"github.com/redhat-partner-solutions/vse-sync-testsuite/pkg/utils"
 )
 
+const (
+	maxConnsecutivePollErrors = 3
```

Comment:
This should be MUCH higher, or even never cause collection to stop. If we are collecting data over days, or power cycling machines, or any number of other things, the collector should include failure to collect in the output but not halt.

Reply:
Fair. Perhaps this is something that should be configurable.

Reply:
For now I'll set it to 1800 which is 30 mins at a rate of 1 per second.